src/components/logview/logview.scss

```diff
@@ -0,0 +1,81 @@
+$message-icon: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUiIGhlaWdodD0iMTUiIHZpZXdCb3g9IjAgMCAxNSAxNSIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0xNCA5LjY2NjY3QzE0IDEwLjQ2NDQgMTMuMzUzMyAxMS4xMTExIDEyLjU1NTYgMTEuMTExMUgzLjg4ODg5TDEgMTRWMi40NDQ0NEMxIDEuNjQ2NyAxLjY0NjcgMSAyLjQ0NDQ0IDFIMTIuNTU1NkMxMy4zNTMzIDEgMTQgMS42NDY3IDE0IDIuNDQ0NDRWOS42NjY2N1oiIHN0cm9rZT0iIzAwQUE0NiIgc3Ryb2tlLXdpZHRoPSIxLjUiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPgo8cGF0aCBkPSJNNCA0LjVIMTAuNzMyMSIgc3Ryb2tlPSIjMDBBQTQ2IiBzdHJva2Utd2lkdGg9IjEuNSIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIi8+CjxwYXRoIGQ9Ik00IDcuNUgxMC43MzIxIiBzdHJva2U9IiMwMEFBNDYiIHN0cm9rZS13aWR0aD0iMS41IiBzdHJva2UtbGluZWNhcD0icm91bmQiLz4KPC9zdmc+Cg==);
```

Comment:
@tyvdh do you know a good way of referencing the actual svg files instead of base64 inlineing them here?